introduction_to_applying_machine_learning/deepar_chicago_traffic_violations/deepar_chicago_traffic_violations.ipynb

```diff
@@ -0,0 +1,467 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using the Amazon SageMaker DeepAR algorithm to forecast traffic violations\n",
+    "_**Using the Amazon SageMaker DeepAR algorithm to predict streets where motorists are most likely to drive above speed limits at different times of the year**_\n",
+    "\n",
+    "---\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "## Contents\n",
+    "\n",
+    "1. [Background](#Background)\n",
+    "1. [Setup](#Setup)\n",
+    "1. [Data](#Data)\n",
+    "1. [Train](#Train)\n",
+    "1. [Host](#Host)\n",
+    "  1. [Evaluate](#Evaluate)\n",
+    "1. [Extensions](#Extensions)\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Background\n",
+    "\n",
+    "This notebook demonstrates time series forecasting using the Amazon SageMaker DeepAR algorithm by analyzing city of Chicago’s red light traffic camera violations. The dataset is hosted by [Data.gov](https://data.gov), and is managed by the [U.S. General Services Administration, Technology Transformation Service](http://www.gsa.gov/portal/category/25729).\n",
+    "\n",
+    "These violations are captured by camera systems and available to improve the lives of public from the [city of Chicago data portal](https://data.cityofchicago.org/). The [Speed Camera Violation  data](https://data.cityofchicago.org/Transportation/Red-Light-Camera-Violations/spqx-js37) set can be used to discern patterns in the data and gain meaningful insights.\n",
```

Comment:
[nitpick]
* "from the city of Chicago data portal" --> "through the city of Chicago data portal"
* "data set" --> "dataset"
  * both versions are used throughout this notebook; I'd do a find-and-replace to make sure you've caught all the instances and made them consistent

Reply:
fixed